docs/admin/authorization.md

```diff
@@ -43,17 +43,25 @@ A request has the following attributes that can be considered for authorization:
   - group (the list of group names the authenticated user is a member of).
```

Comment:
The attributes described here are actually generic to all authorization, not just ABAC.